packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts

```diff
@@ -8,11 +8,11 @@ import { WorkflowTarget } from '../../../types';
 
 const MAX_LABEL_LENGTH = 45;
 
-export enum SubscriptionTabIds {
-    GENERAL_TAB = 'general-id',
-    SERVICE_CONFIGURATION_TAB = 'service-configuration-id',
-    PROCESSES_TAB = 'processes-id',
-    RELATED_SUBSCRIPTIONS_TAB = 'related-subscriptions-id',
+export enum WfoSubscriptionDetailTab {
```

Comment:
I would say to only name components Wfo..Name not the types and enums -- except props

Reply:
Renamed